datastructures/src/main/java/com/aparamonov/algostructure/list/ListUsage.java

```diff
@@ -0,0 +1,145 @@
+package com.aparamonov.algostructure.list;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.*;
+import java.util.stream.Stream;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+
+public class ListUsage {
+
+    public static void main(String[] args) {
```

Comment:
i would avoid this interactivity at all since we have possibility to run unit tests at any desired moment of time
please consider using Travis CI (https://travis-ci.org/)